src/query/api/v1/handler/namespace/common.go

```diff
@@ -72,7 +72,7 @@ type Handler struct {
 	instrumentOpts instrument.Options
 }
 
-type AddRouteFn func(path string, handler http.Handler, methods ...string)
+type addRouteFn func(path string, handler http.Handler, methods ...string)
```

Comment:
nit: rather than defining this per package, can we have it be defined in a single package and imported in each? (May need to make a utils package if there's circular dependencies)